### ISSUE TITLE
ImportError: Could not load shared library "libmseed_Windows_64bit_py27.pyd".

### ISSUE THREAD

**Issue by nicobastias** [obspy/obspy#1898]
Hi everyone!,

My obspy worked very well, but today when try to import obspy 

`from obspy.core import read
`

display this error message:

```
ImportError: Could not load shared library "libmseed_Windows_64bit_py27.pyd".

 LoadLibrary() argument 1 must be string, not unicode
```

My python version is 2.7.12 and my obspy version is 1.0.2, I'm install obspy using Anaconda on Windows 10. My python programing skills are low, so this bug may be really easy to solve. Thanks, in advance

**Comment by krischer** [comment]
This should already be fixed in ObsPy 1.0.3 so if you update it should be gone.

https://github.com/obspy/obspy/pull/1636

Please close this issue if it fixes your problem.

**Comment by megies** [comment]
I'm pretty sure that' exactly the above issue, so I'm closing this. Please feel free to reopen if the problem remains after updating to 1.0.3.
